hrui/CRM/HRUI/Upgrader/Steps/4706.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+trait CRM_HRUI_Upgrader_Steps_4706 {
+
+  /**
+   * Adds "is applying for NI/SSN" field
+   */
+  public function upgrade_4706() {
+    $groupName = 'Inline_Custom_Data';
+    $customGroup = civicrm_api3('CustomGroup', 'get', ['name' => $groupName]);
+
+    if ($customGroup['count'] != 1) {
+      throw new \Exception("Cannot continue without Inline_Custom_Data group");
+    }
+
+    $customGroup = array_shift($customGroup['values']);
+
+    // Add "is applying" field
```

Comment:
I see the comment here indicating that the field to be added will be in present continuous tense, however it is in past tense. i.e `has_applied_for_identification` . Also saw on the PR overview that a new field is added which shows  that contact is in process of applying for SSN.
Is this right?